internal/server/assets/fs.go

```diff
@@ -0,0 +1,6 @@
+package assets
+
+import "embed"
+
+//go:embed *
```

Comment:
Does that also embed `fs.go`? I'd prefer to move the go file to the parent folder.

Reply:
Yes it does, I've changed it to only include `*.html` and `*.css`.

Reply:
Yeah, we can move it there. I just needed to get rid of the `assets/` path component, but there's https://pkg.go.dev/io/fs#Sub